pysat/utils/io.py

```diff
@@ -500,9 +990,9 @@ def load_netcdf_pandas(fnames, strict_meta=False, file_format='NETCDF4',
                     obj_key = dim[0]
                 else:
                     estr = ''.join(['Epoch label: "', epoch_name, '"',
-                                    ' was not found in loaded dimensions [',
+                                    ' not found in loaded dimensions [',
                                     ', '.join(dim), ']'])
-                    raise KeyError(estr)
+                    raise ValueError(estr)
```

Comment:
I don't like the new wording, keeping "was" seems to create an easier to read sentence.  Also curios as to the motivation behind the Key->Value change.

Reply:
For consistency with another equivalent error, in xarray I think. Based on the pysatModels SAMI test looks like I went the wrong way. 🙈 I'll spin up on this and the last remaining comment way up there about a None here shortly (today).

Reply:
To close the loop, Errors were updated for consistency with the existing statement. The SAMI test error was something else. Fix incorporated into pysatModels develop.